instrumentation/src/main/java/io/opentelemetry/android/instrumentation/crash/CrashReporter.java

```diff
@@ -38,16 +45,47 @@ public void installOn(InstrumentedApplication instrumentedApplication) {
                 Thread.getDefaultUncaughtExceptionHandler();
         Thread.setDefaultUncaughtExceptionHandler(
                 new CrashReportingExceptionHandler(
-                        buildInstrumenter(instrumentedApplication.getOpenTelemetrySdk()),
-                        instrumentedApplication.getOpenTelemetrySdk().getSdkTracerProvider(),
+                        buildInstrumenter(
+                                instrumentedApplication
+                                        .getOpenTelemetrySdk()
+                                        .getSdkLoggerProvider()),
+                        instrumentedApplication.getOpenTelemetrySdk().getSdkLoggerProvider(),
                         existingHandler));
     }
 
-    private Instrumenter<CrashDetails, Void> buildInstrumenter(OpenTelemetry openTelemetry) {
-        return Instrumenter.<CrashDetails, Void>builder(
-                        openTelemetry, "io.opentelemetry.crash", CrashDetails::spanName)
-                .addAttributesExtractor(new CrashDetailsAttributesExtractor())
-                .addAttributesExtractors(additionalExtractors)
-                .buildInstrumenter();
+    private void emitCrashEvent(Logger crashReporter, CrashDetails crashDetails) {
+        Throwable throwable = crashDetails.getCause();
+        Thread thread = crashDetails.getThread();
+        AttributesBuilder attributesBuilder =
+                Attributes.builder()
+                        .put(SemanticAttributes.EXCEPTION_ESCAPED, true)
+                        .put(SemanticAttributes.THREAD_ID, thread.getId())
+                        .put(SemanticAttributes.THREAD_NAME, thread.getName())
```

Comment:
Maybe worth considering adding a new attribute if it is the main thread or not

Reply:
Hmm, I'm actually not sure if the concept of "main thread" is as relevant in every other platform as it is in Android in order to have it as a general thread attr, but I guess we can always try and check what people think about it. If you have some use cases in mind that would benefit from this attr, I think you could mention them in an issue in [this repo](https://github.com/open-telemetry/semantic-conventions), or maybe just create a draft adding the new attr [in this file](https://github.com/open-telemetry/semantic-conventions/blob/0b437c0edf3221786cb516c7f24c526d2a39a6a2/docs/attributes-registry/thread.md), is usually easier to convey ideas with code changes rather than only with a description in an issue.

Reply:
good point, will do, just for context, exception triage, IMO main thread errors are most likely more important than background errors (if the thread is a daemon) maybe the app won't even crash, but not sure if in this case the default signal handler would be called anyway.